### PR TITLE
fix: opens external anchor using rel=noopener

### PR DESCRIPTION
**Pull request by JLHwung**
see https://developers.google.com/web/tools/lighthouse/audits/noopener

Thank you for creating a pull request to contribute to Hexo code! Before you open the request please review the following guidelines and tips to help it be more easily integrated:

- [-] Add test cases for the changes.
- [x] Passed the CI test.

### CODE REVIEW

**Comment by JLHwung** [comment]
Any suggestions? cc @NoahDragon

**Comment by NoahDragon** [comment]
Sorry, I added review several days ago but forgot to submit them.

File: lib/plugins/helper/link_to.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function linkToHelper(path, text, options) {
 
   if (attrs.external) {
     attrs.target = '_blank';
-    attrs.rel = 'external';
+    attrs.rel = 'noopener';
```
[the remark this answers is not in the document]
Comment (JLHwung) [reply lib/plugins/helper/link_to.js]: Good to go, I will rework on this PR.

**Comment by JLHwung** [comment]
LGTM, @NoahDragon I suggest we publish a new patch version with latest hexo-log or bunyan-related dependencies.

**Comment by NoahDragon** [comment]
@JLHwung Agree.

**Pull request merged**